ui/src/components/EventConfiguration/Modal/DeleteEventConfigSourceModal.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+  <div class="delete-event-config-source-modal">
+    <FeatherDialog
+      v-model="store.deleteEventConfigSourceModalState.visible"
+      :labels="labels"
+      hide-close
+      @hidden="store.hideDeleteEventConfigSourceModal()"
+    >
+      <div class="modal-body">
+        <p>
+          This will delete the event configuration source:
+          <strong>{{ store.deleteEventConfigSourceModalState.eventConfigSource?.filename }}</strong>
+        </p>
+        <p>
+          <strong>Note:</strong> This EventConf source has
+          <strong>{{ store.deleteEventConfigSourceModalState.eventConfigSource?.eventCount }}</strong> events associated
+          with it and will be deleted.
+        </p>
+        <p><strong>Are you sure you want to proceed?</strong></p>
+      </div>
+      <template v-slot:footer>
+        <FeatherButton @click="store.hideDeleteEventConfigSourceModal()"> Cancel </FeatherButton>
+        <FeatherButton
+          primary
+          @click="deleteEventConfigSource()"
+        >
+          Delete
+        </FeatherButton>
+      </template>
+    </FeatherDialog>
+  </div>
+</template>
+
+<script lang="ts" setup>
+import { useEventConfigStore } from '@/stores/eventConfigStore'
+import { FeatherButton } from '@featherds/button'
+import { FeatherDialog } from '@featherds/dialog'
+
+const store = useEventConfigStore()
+const labels = {
+  title: 'Delete EventConf Source'
+}
+
+const deleteEventConfigSource = async () => {
+  try {
+    // Call API to delete the event configuration source
```

Comment:
Maybe add a `TODO` here, so it is clear we are waiting for the Rest API implementation.